### ISSUE TITLE
Fix for publishing CI

### ISSUE THREAD

**Issue by matthewfeickert** [eschanet/simplify#3]
The publishing CI is currently failing as

https://github.com/eschanet/simplify/blob/22d3a0372d539d6a9354508442e607fbe1601671/.github/workflows/publish.yml#L43

is checking for a library name of `simplify` but the library is named `simplify-hep` and so

```
python -m build --sdist --wheel --outdir dist/ .
```

gives
```
$ tree dist/
dist/
├── simplify_hep-0.1.1-py3-none-any.whl
└── simplify-hep-0.1.1.tar.gz

0 directories, 2 files
```

The quick fix is to just change the name in CI to `simplify-hep`, but this brings up the question of the name.

I see that as there is already https://pypi.org/project/simplify/ you probably renamed the library to `simplify-hep`. That library hasn't been touched since 2013 so I would contact the PyPA and ask to be given control of the library, as the author isn't maintaining it. If you don't want to go down that route though I'd suggest renaming everything to be consistent (e.g., move everything to be named `simplify-hep`), but I think `simplify` is a better name so I'd just email the PyPA. :+1:

**Comment by eschanet** [comment]
Hi @matthewfeickert, this is again a good point. I also prefer `simplify` as a name, so I went ahead and opened pypa/pypi-support#770.

**Comment by eschanet** [comment]
Accidentally closed, #5 is only a temporary fix.

**Comment by matthewfeickert** [comment]
Resolved by 4a334b4c4158ef67d9dd97f8d3cd352da859892b

(@eschanet if you think there's anything else that needs to be accounted for here feel free to reopen)
